resources/js/processes/components/CreateProcessModal.vue

```diff
@@ -286,7 +287,9 @@
             const url = `/package-ai/processes/create/${response.data.id}`;
             this.$emit("process-created-from-modeler", url, response.data.id, response.data.name);
           } else {
-            window.location = `/modeler/${response.data.id}`;
+            this.isAiGenerated
+              ? window.location = "/modeler/" + response.data.id + "?ai=1"
```

Comment:
instead of one, it can be of Boolean value. Although 1 means true , but you get what I mean